.github/workflows/gradle_publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Gradle Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Create local.properties with a read token
+      run: |
+        # Use a personal read token to install the Cyface Utils package
+        echo "github.user=${{ secrets.GITHUB_READ_ACCOUNT }}" > local.properties
+        echo "github.token=${{ secrets.GITHUB_READ_TOKEN }}" >> local.properties
```

Comment:
Unfortunately, there is currently no cleaner way than using a personal access token to install dependencies from the Github Registry from within the Github Actions. See tweet from Github Product Manager: https://twitter.com/char_fish/status/1191442780729556993?s=21

Fyi:
- Currently authentication is always required when *downloading* dependencies from the Github Registry, no matter whether the dependency project is open-source or private.
- The token with read-only access is stored in the Github Secrets. Those are only accessible from Github Actions and not in PR builds, i.e. the token should be safe.